### ISSUE TITLE
version of kombu needs to be updated for newer python

### ISSUE THREAD

**Issue by adenbley** [miguelgrinberg/flask-celery-example#8]
python 2.7.11 does not work with the version of kombu in `requriements.txt`.

**Comment by scarroll32** [comment]
Update kombu to at least 3.0.30 in **requirements/dev.txt**

```
kombu==3.0.30
```

**Comment by ernstki** [comment]
Just to add a bit of useful context (hopefully), [here is the (now-closed) Kombu issue affecting Python 2.7.11](https://github.com/celery/kombu/issues/545) and this appears to be the backtrace you get at `kombu==3.0.24` which goes away after following @seanfcarroll's advice, above:

```
$ python app.py 
Traceback (most recent call last):
  File "app.py", line 7, in <module>
    from celery import Celery
  File "/Users/kernst/devel/MLU/GrosseLab/flask-celery-example/venv/lib/python2.7/site-packages/celery/__init__.py", line 130, in <module>
    from celery import five
  File "/Users/kernst/devel/MLU/GrosseLab/flask-celery-example/venv/lib/python2.7/site-packages/celery/five.py", line 149, in <module>
    from kombu.utils.compat import OrderedDict  # noqa
  File "/Users/kernst/devel/MLU/GrosseLab/flask-celery-example/venv/lib/python2.7/site-packages/kombu/utils/__init__.py", line 19, in <module>
    from uuid import UUID, uuid4 as _uuid4, _uuid_generate_random
ImportError: cannot import name _uuid_generate_random
```

It might be helpful to change the issue title to be/include the error message `ImportError: cannot import name _uuid_generate_random`.
